src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -129,28 +158,22 @@ The data you can expect back for both the initial grant code exchange and subseq
 }
 ```
 
-#### How to use for requests
+### Auth Tokens
 
-When making requests to the Sentry API, you use the token just like you would when you're typically making [API requests](/api/auth/). Tokens are associated with the installation, meaning they have access to the Sentry organization that installed your integration.
+Auth Tokens are referred to as _tokens_ in the Integration Platform. Auth tokens are passed using an auth header and are used to authenticate as a user account with the API. In the case of the Integration Platform, these tokens are used to authenticate as an application (with access to a specific organization) instead of a user.
 
-#### Expiration
 
-Tokens expire every eight hours.
+### Using Tokens for Requests
 
-#### Verifying Installations (optional)
+When making requests to the Sentry API, you use the token just like you would when you're making typical [API requests](/api/auth/). Tokens are associated with the installation, meaning they have access to the Sentry organization that installed your integration.
 
-Typically if you have the Redirect URL configured, there is work happening on your end to 'finalize' the installation. If this is the case, we recommend enabling the Verify Install option for your integration. Once enabled, you will need to send a request marking the installation as officially 'installed.'
+### Token Expiration
 
-```python
-requests.put(
-    u'https://sentry.io/api/0/sentry-app-installations/{}/'.format(install_id),
-    json={'status': 'installed'},
-)
-```
+Tokens expire every eight hours.
```

Comment:
In case someone searches for this, it's good to be specific here since this answer is different for internal integrations. 

```suggestion
Tokens for public integrations expire every eight hours.
```